test/libsolidity/util/compiler/Compiler.cpp

```diff
@@ -0,0 +1,62 @@
+#include <test/libsolidity/util/compiler/Compiler.h>
+
+using namespace solidity;
+using namespace solidity::frontend::test;
+
+CompiledContract const* CompilerOutput::contract(ContractName const& _name) const
+{
+    auto const& sourceName = _name.source();
+    auto const& contractName = _name.contract();
+
+    if (auto source = m_sourceUnits.find(sourceName); source != m_sourceUnits.end())
```

Comment:
```suggestion
    if (
        auto const source = m_sourceUnits.find(sourceName); 
        source != m_sourceUnits.end()
    )
```

This is the generally preferred style for if-with-initializer throughout the codebase :)